src/spas/libs/member-management/feature/tsconfig.json

```diff
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "types": ["node", "jest"]
   },
-  "include": ["**/*.ts"]
+  "include": ["**/*.ts", "../domain/src/lib/mocks/mock-users.ts", "../domain/src/lib/models/user-profile.ts", "../domain/src/lib/user.mock.service.ts", "../domain/src/lib/user.service.spec.ts"]
```

Comment:
was there a reason why specific .ts files were added here? **/*.ts should cover it. if you remove ```, "../domain/src/lib/mocks/mock-users.ts", "../domain/src/lib/models/user-profile.ts", "../domain/src/lib/user.mock.service.ts", "../domain/src/lib/user.service.spec.ts"``` does the feature lib still compile?

Reply:
Not sure, I believe the CLI did this automatically.  But it does still compile after removing those .ts files.

Reply:
🆒